controllers/metering/controllers/metering_controller.go

```diff
@@ -20,210 +20,238 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"strconv"
 	"time"
 
+	"k8s.io/client-go/util/retry"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/types"
+
 	"github.com/go-logr/logr"
 	infrav1 "github.com/labring/sealos/controllers/infra/api/v1"
 	userv1 "github.com/labring/sealos/controllers/user/api/v1"
-	"k8s.io/apimachinery/pkg/api/resource"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	meteringv1 "github.com/labring/sealos/controllers/metering/api/v1"
 
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
-var SealosSystemNamespace string
-
-func init() {
-	SealosSystemNamespace = os.Getenv("SEALOS_SYSTEM_NAMESPACE")
-}
-
 const (
-	FinalizerName          = "metering.sealos.io/finalizer"
-	UserAnnotationOwnerKey = "user.sealos.io/creator"
+	MeteringPrefix           = "metering-"
+	ResourceQuotaPrefix      = "quota-"
+	METERINGNAMESPACEENV     = "METERING_SYSTEM_NAMESPACE"
+	DEFAULTMETERINGNAMESPACE = "metering-system"
 )
```

Comment:
place these consts in to api/common_types.go is batter?

Reply:
and notice spile format.

Reply:
if METERINGNAMESPACEENV will not change, switch cr metering to cluster scpoe is ok?

Reply:
> place these consts in to api/common_types.go is batter?

i agree

Reply:
> if METERINGNAMESPACEENV will not change, switch cr metering to cluster scpoe is ok?

What is the purpose switch cr metering to cluster scpoe